docs/release_notes.rst

```diff
@@ -14,6 +14,11 @@ EIA 930
 * Added EIA 930 hourly data through the end of October as part of the Q3 quarterly
   release. See :issue:`3942` and :pr:`3946`.
 
+EIA 923
+~~~~~~~
+* Added EIA 923 data from August 2024 as part of the Q3 quarterly release.
```

Comment:
Just from August, or up through August? Is this because we just updated 923 data when we did the final data release?

Reply:
Yep